todo/middlewares/jwt_auth.py

```
@@ -0,0 +1,41 @@
+from django.conf import settings
```

Pylint (3.3.7)

[convention] Missing module docstring

(C0114)